Digipost.Signature.Api.Client.Archive.Tests/Smoke/TestHelper.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Digipost.Signature.Api.Client.Core.Exceptions;
+using Digipost.Signature.Api.Client.Core.Tests.Smoke;
+using Xunit;
+
+namespace Digipost.Signature.Api.Client.Archive.Tests.Smoke
+{
+    public class TestHelper : TestHelperBase
+    {
+        private readonly ArchiveClient _archiveClient;
+        private readonly DocumentOwner _documentOwner;
+        private readonly ArchiveDocument _document;
+        public TestHelper(ArchiveClient archiveClient)
+        {
+            _archiveClient = archiveClient;
+            _documentOwner = new DocumentOwner(_archiveClient.ClientConfiguration.GlobalSender.OrganizationNumber);
+            _document = new ArchiveDocument("1234");
+        }
+        
+        public TestHelper Get_PAdES()
+        {
+            var pades = _archiveClient.GetPades(_documentOwner, _document).Result;
+            Assert.True(pades.CanRead);
+            return this;
+        }
+
+        public void Download_pades_and_expect_client_error()
+        {
+            try
+            {
+                Get_PAdES();
+            }
+            catch (AggregateException ex)
+            {
+                if(ex.InnerException is UnexpectedResponseException)
+                {
+                    UnexpectedResponseException exception = (UnexpectedResponseException) ex.InnerException;
+                    Assert.Equal("ARCHIVED_DOCUMENT_NOT_FOUND", exception.Error.Code);
```

Comment:
Jeg tenkte på noe, @manuelAlonsoPerez. Kanskje det er litt vanskelig for en avsender å sjekke om `InnerException` er denne typen og har en error code. Kanskje vi kan gjøre en `try/catch` i `ArchiveClient` som fanger akkurat denne og lager en ny exception, f.eks `DocumentNotFoundInArchiveException` som har en god feilmelding?

Reply:
Høres ut som en god idé. Notér det bak øret 👂, og ta med det før final release.